docs/content/tutorials/new-app/index.md

```diff
@@ -0,0 +1,378 @@
+
+## Prerequisites
+
+- [Supported Kubernetes cluster]({{< ref "/guides/operations/kubernetes/overview" >}})
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-radius-bicep-vs-code-extension" >}})
+
+## Step 1: Initialize a Radius Environment and Application
+
+[Radius Environments]({{< ref "/guides/deploy-apps/environments/overview" >}}) are where applications are deployed. Environments determine how an application runs on a particular platform (_like AWS or Azure_).
+
+1. Begin by creating a new directory for your application:
+
+   ```bash
+   mkdir myapp
+   cd myapp
+   ```
+
+1. Initialize a new Radius Environment with [`rad init`]({{< ref rad_init >}}):
+   
+   ```bash
+   rad init
+   ```
+
+   When asked if you want to create a new application select "Yes". This will create a new file named `app.bicep` in your directory where your application will be defined.
+
+   {{< alert title="💡 Development Environments" color="info" >}}
+   By default `rad init` gets you up and running with a development-focused environment where most of the environment configuration is handled for you, including Recipes (_more on that soon_). If you would like to fully    customize your environment, you can run `rad init --full`
+   {{< /alert >}}
+
+## Step 2: Inspect your Application
+
+Radius Applications are where all your app's resources and relationships come together. Let's take a look at this Radius Application to learn more about it.
+
+1. View the full Application definition by running [`rad app show -o json`]({{< ref rad_application_show >}}):
+
+   ```bash
+   rad app show myapp -o json
+   ```
+
+   You will see the full App definition in its raw JSON format:
+   
+   ```
+   {
+     "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/applications/myapp",
+     "location": "global",
+     "name": "myapp",
+     "properties": {
+       "environment": "/planes/radius/local/resourceGroups/default/providers/Applications.Core/environments/default",
+       "provisioningState": "Succeeded",
+       "status": {
+         "compute": {
+           "kind": "kubernetes",
+           "namespace": "default-myapp"
+         }
+       }
+     },
+     "systemData": {},
+     "tags": {},
+     "type": "Applications.Core/applications"
+   }
+   ```
+   
+   There are a few important things to note about the application definition:
+
+   - **`id`** is the fully-qualified UCP resource ID of the application. This value is used to uniquely identify the application in the Radius system.
+   - **`location`** is the location of the application. For now, all applications are deployed to the `global` location.
+   - **`environment`** specifies where the application will be deployed. In this case, the application will be deployed into the `default` environment that was created by `rad init`.
```

Comment:
It would help to more clearly distinguish the definition of both location and environment.